package.json

```diff
@@ -44,11 +44,11 @@
     ]
   },
   "dependencies": {
-    "js-search": "^1.3.1",
-    "react-select": "^1.0.0-beta14"
+    "js-search": "^1.3.1"
   },
   "peerDependencies": {
-    "react": "^0.14.0 || ^15.0.0"
+    "react": "^0.14.0 || ^15.0.0",
+    "react-select": "^1.0.0-beta14"
```

Comment:
You don't use `react-select` directly in your module, so I see no reason use it as a dependency.